client/src/pages/ContactsPage.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Contacts from "../components/Contacts";
+import Footer from "../components/Footer";
+import HeaderBrown from "../components/HeaderBrown";
+
+const ContactsPage = () => {
+  return (
+    <>
+      <HeaderBrown />
+      <Contacts />
+      <Footer />
```

Comment:
This looks repetitive and could be extracted to some kind of `MainLayout` component.

Reply:
We will make it